apps/organizations/views.py

```diff
@@ -0,0 +1,95 @@
+# Django Imports
+from django.contrib.auth.decorators import login_required
+from django.http import Http404
+from django.shortcuts import get_object_or_404
+from django.utils import timezone
+from django.utils.decorators import method_decorator
+from django.views import View
+
+# HTK Imports
+from htk.apps.organizations.enums import OrganizationMemberRoles
+from htk.utils import (
+    htk_setting,
+    resolve_model_dynamically,
+)
+from htk.view_helpers import render_custom as _r
+from htk.view_helpers import wrap_data
```

Comment:
I think there is a way to get a dynamic `wrap_data` function from `settings.py`

Reply:
Done used `HTK_VIEW_CONTEXT_GENERATOR` setting